targets/targets.json

```diff
@@ -345,6 +345,16 @@
         "mbed_ram1_start": "0x2007c000",
         "mbed_ram1_size": "0x8000",
         "bootloader_supported": true,
+        "sectors": [
+            [
+                0,
+                4096
+            ],
+            [
+                65536,
+                32768
+            ]
+        ],
```

Comment:
How is this change tested in CI? What tests would fail in our CI if we got these sector values incorrect?

Reply:
If I am not wrong, currently, we don't have any test available to test the sector values.